cmd/celestia-appd/cmd/root.go

```diff
@@ -95,7 +95,7 @@ func NewRootCmd() *cobra.Command {
 			tmCfg.Consensus.TimeoutPropose = appconsts.TimeoutPropose
 			tmCfg.Consensus.TargetHeightDuration = appconsts.TargetHeightDuration
 			tmCfg.Consensus.SkipTimeoutCommit = false
-			tmCfg.TxIndex.Indexer = "null"
+			tmCfg.TxIndex.Indexer = "kv"
```

Comment:
[optional] since `kv` is the default, I verified that we could also remove this line entirely

```suggestion
```

and the generated `config.toml` will have

```toml
indexer = "kv"
```